models/mace/test_mace.py

```diff
@@ -129,35 +130,39 @@
 }[ase_filter]
 optim_cls: Callable[..., Optimizer] = {"FIRE": FIRE, "LBFGS": LBFGS}[ase_optimizer]
 
-for atoms in tqdm(atoms_list, desc="Relaxing"):
+for atoms in tqdm(deepcopy(atoms_list), desc="Relaxing"):
```

Comment:
is this deliberate or left over from `BatchOptimizer`?

Reply:
deliberate in order to be able to repeatedly run the optimization in order to debug as if not the original list is mutated in place. It could be excluded from this PR if wanted but I think the isolation is fairly helpful